static/js/apps/biomed_nl/app.tsx

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @jsxImportSource @emotion/react */
+
+/**
+ * Main component for the Biomedical NL interface page
+ */
+
+import { css, ThemeProvider } from "@emotion/react";
+import axios from "axios";
+import _ from "lodash";
+import queryString from "query-string";
+import React, { ReactElement, useEffect, useState } from "react";
+import Collapsible from "react-collapsible";
+import ReactMarkdown from "react-markdown";
+import { Input } from "reactstrap";
+import rehypeRaw from "rehype-raw";
+import remarkGfm from "remark-gfm";
+
+import theme from "../../theme/theme";
+import { stringifyFn } from "../../utils/axios";
+import { updateHash } from "../../utils/url_utils";
+
+// Constants for URL hash parameters
+const URL_HASH_PARAMS = {
+  q: "q",
+};
+
+// Interface for the response received from Biomed NL API.
+interface BiomedNlApiResponse {
+  answer: string;
+  debug: string;
+}
+
+// Interface for the displayed answer.
+interface DisplayedAnswer {
+  answer: string;
+  footnotes: string;
```

Comment:
curious, are there footnotes in the biomed answer?

Reply:
there might be depending on how the references are formatted in the final version. I'll keep this for now and remove it if it's not necessary in the follow up PR -- if that seems reasonable to you?

Reply:
that works!